tests/HttpExtractorTest.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Keboola\HttpExtractor;
+
+use function file_get_contents;
+use GuzzleHttp\Client;
+use GuzzleHttp\Handler\MockHandler;
+use GuzzleHttp\HandlerStack;
+use GuzzleHttp\Middleware;
+use GuzzleHttp\Psr7\Request;
+use GuzzleHttp\Psr7\Response;
+use GuzzleHttp\Psr7\Uri;
+use PHPUnit\Framework\TestCase;
+use function sys_get_temp_dir;
+
+class HttpExtractorTest extends TestCase
+{
+    /** @var mixed[] */
+    private $history = [];
+
+    public function testExtract(): void
```

Comment:
Přidal bych sem ještě test na 404, že má skončit user exception a nějakou smysluplnou hláškou na stdout pro endusera.